M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -653,17 +670,20 @@ doc ///
 	     L = randomAbstractSimplicialComplex(6,3)
 	  Text
 	     Create the random complex $Y_d(n,m)$ which has vertex set
-             $[n]$ and $(d − 1)$-skeleton, and has $m$ $d$-dimensional faces,
+             $[n]$ and complete $(d − 1)$-skeleton, and has exactly m d-dimensional faces,
```

Comment:
I think this line was accidentally reverted.

Reply:
I think this line still needs to be fixed.  The "exactly" was removed since there might be fewer than m d-dimensional faces (unless the new `Verify` option is set to true), but now it's back.